docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<p>The terms "train" and "test" are used here so to remain faithful to the paper. However, personally speaking, the term "train"  may not make sense in offline setting. (It may make sense online setting but we have to wait and read the corresponding part in the paper later)</p><p> </p><p>The paper introduces the term "split index"  which is the index that divides the time series <code>T</code> into two parts: train part <code>T[: split_index] </code> and test part <code>T[split_index:]</code> Then, to detect whether the subsequence <code>S_i = T[i:i+m]</code> is discord or not, all of its left neighbors upto index <code>i </code> are considered. In other words, BOTH train and test parts are considered in the decision making process for the subsequence  <code>S_i</code>.  So, it feels a little bit strange to use the terms "train" and "test"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>